tensorflow/lite/micro/docs/arm.md

```diff
@@ -0,0 +1,47 @@
+<!--ts-->
+* [Arm(R) IP support in Tensorflow Lite for Microcontrollers (TFLM)](#arm-ip)
+   * [Arm(R) Cortex(R)-M processor family](#cortex-m)
+   * [CMSIS-NN optimized library](#cmsis-nn)
+   * [Arm(R) Ethos(TM)-U microNPU family](#ethos-u)
+   * [Arm(R) Corstone(TM)-300 FVP](#corstone-300)
+<!--te-->
+
+# Arm(R) IP support in Tensorflow Lite for Microcontrollers (TFLM)
+
+This doc outlines how to use Arm IP with TFLM. The following sub chapters
+contain more details of the respective IP.
+
+## Arm(R) Cortex(R)-M processor family
+Arm's Cortex-M processor support is fully integrated to TFLM. To build a TFLM
+library for any Cortex-M processor, check out the [Cortex-M generic readme](https://github.com/tensorflow/tflite-micro/tree/main/tensorflow/lite/micro/cortex_m_generic/README.md).
+Additionally, CMSIS-NN provides optimal performance executing machine learning
+workloads on Cortex-M. See the [sub chapter CMSIS-NN](#cmsis-nn).
+
+
+## CMSIS-NN optimized library
+Common Microcontroller Software Interface Standard for Neural Networks
+(CMSIS-NN) is a collection of efficient neural network kernels developed to
+maximize performance on Cortex-M processors. The CMSIS-NN optimized kernel are
+highly integrated to TFLM. For more information how to utilize these kernels,
+see [CMSIS-NN readme](https://github.com/tensorflow/tflite-micro/tree/main/tensorflow/lite/micro/kernels/cmsis_nn/README.md).
+
+
+## Arm(R) Ethos(TM)-U microNPU family
+The Ethos-U microNPU (Neural Processing Unit) family consist of [Ethos-U55](https://www.arm.com/products/silicon-ip-cpu/ethos/ethos-u55)
+and [Ethos-U65](https://www.arm.com/products/silicon-ip-cpu/ethos/ethos-u65).
+Ethos-U55 is designed to accelerate ML inference in area-constrained embedded
+and IoT devices, whereas Ethos-U65 extends its applicability to Arm Cortex-A,
+Cortex-R and Neoverse-based systems.
```

Comment:
I feel it may be better to remove Line 32 to Line 34 just because the reference to Corext-A/R/Neoverse-based may lead people to ask for CMSIS-NN optimization on those platforms which I think neither side has a solid plan yet.

Reply:
Will do.

Reply:
I added a clarification. This particular sub-chapter is for Ethos-U products. CMSIS-NN is handled in the subchapter before.